Dockerfile

```diff
@@ -78,14 +78,15 @@ WORKDIR ${APP_PATH}
 COPY ./Gemfile ./Gemfile.lock ./
 RUN bundle config set --local with "${RAILS_ENV}" && \
     bundle config set --local without "${BUNDLE_WITHOUT}" && \
+    bundle config set --local path "${BUNDLE_PATH}" && \
     bundle install && \
     gem install aws-sdk-s3 && \
     rm -rf ${BUNDLE_PATH}/cache/*.gem && \
     find ${BUNDLE_PATH}/ -name "*.c" -delete && \
     find ${BUNDLE_PATH}/ -name "*.o" -delete
 
 COPY . .
-RUN bundle exec rake assets:precompile && rm .env
+RUN rm .env
```

Comment:
Shinwoo says:

> Originally for the blacklight-cornell container, the asset precompile was part of the container startup script.
> What ended up happening was that that part took too long and AWS thought something was wrong and kept killing the deployment process.
> That's why we precompile the assets as build process for blacklight-cornell.

TODO: Christina will compare deploy times with precompile step in Dockerfile vs entrypoint script. Current deploy times on this branch with the precompile step in run.sh are 4-10 minutes (typically ~5 minutes).

Reply:
Asset compilation step in entrypoint script:
- 1st deploy:
    - Jenkins deploy job (image build & push): 21 sec
    - ECS task deploy: 4 minutes, 16 seconds
- 4th deploy:
    - Jenkins deploy job (image build & push): 21 sec
    - ECS task deploy: 4 minutes, 10 seconds
Asset compilation step in Dockerfile:
- 2nd deploy:
    - Jenkins deploy job (image build & push): 59 sec
    - ECS task deploy: 4 minutes
- 3rd deploy: 
    - Jenkins deploy job (image build & push): 58 sec
    - ECS task deploy: 4 minutes, 44 seconds

When asset compilation step is in Dockerfile, the Jenkins deploy job is somewhat longer, but the ecs task deploy length doesn't seem to have much of a pattern based on only a handful of test deploys. Seems like whether this is in Dockerfile or the entrypoint script doesn't make a huge difference, so I'm going to keep it in the entrypoint script for the web server container since the sidekiq container doesn't need it.